docs/serving/samples/cloudevents/cloudevents-rust/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use actix_web::{post, web, App, HttpRequest, HttpServer};
+
+#[post("/")]
+async fn post_event(req: HttpRequest, payload: web::Payload) -> Result<String, actix_web::Error> {
```

Comment:
I think that this is the Request/Respond flow (Broker-style), which is awesome.  Could you also include the `K_SINK` style and have them mutate the event to showcase that?

Reply:
Can we do it in a separate PR? For now i made a bare minimum example, i plan to add more later. I still need to figure out some things in the sdk before :smile:

Reply:
[this](https://github.com/knative/docs/pull/2480#discussion_r427685504) comment tries to elaborate on the motivations a bit, and my preference is to bias towards consistency across these samples as much as possible so that they can serve as per-language drop-in replacements for:
1. How to use Broker
2. How to write a source (and use Sink Binding)

Reply:
Ok I'll try to find out something :smile: